theano/gpuarray/opt.py

```diff
@@ -328,8 +607,8 @@ def local_gpua_alloc_empty_to_zeros(node):
     if isinstance(node.op, GpuAllocEmpty):
         context_name = infer_context_name(*node.inputs)
         z = numpy.asarray(0, dtype=node.outputs[0].dtype)
-        return [GpuAlloc()(as_gpuarray_variable(z, context_name),
-                           *node.inputs)]
+        return [gpu_alloc(None)(as_gpuarray_variable(z, context_name),
```

Comment:
Why use `None` as the context, instead of the `context_name` you inferred earlier?